server/sonar-server/src/test/java/org/sonar/ce/log/CeLoggingTest.java

```diff
@@ -65,15 +42,8 @@
   public TemporaryFolder temp = new TemporaryFolder();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused "TemporaryFolder". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2924)